agent.go

```diff
@@ -437,17 +438,10 @@ func (s *sandbox) initChannel() error {
 
 	s.channel = c
 
-	return s.channel.setup()
+	return nil
 }
 
 func (s *sandbox) startGRPC() error {
```

Comment:
with this patch now `startGRPC` always returns nil, wouldn't be better return nothing

Reply:
good point!